man/resolved.conf.xml

```diff
@@ -269,6 +269,18 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStubListenerExtra=</varname></term>
+        <listitem><para>Takes an IPv4 or IPv6 address to listen on, optionally prefixed by <literal>:</literal> protocol,
+        <literal>udp</literal> or <literal>tcp</literal> and suffixed by a <literal>:</literal>
+        followed by a numeric port number. May be used multiple times. If the prefix (protocol) is missing, then listen on both
```

Comment:
`Takes an IPv4 or IPv6 address to listen on. The address may optionally be prefixed by <literal>:</literal> and a protocol name (<literal>udp</literal> or <literal>tcp</literal>). It may also be optionally suffixed by <literal>:</literal> and a numeric port number. If the protocol is not specified, the service will listen on both IPv4 and IPv6. This option may be used multiple times. Note that this is independent of the primary DNS stub configured with <varname>DnsStubListener=</varname>, and only configures <emphasis>additional</emphasis> sockets to listen on. Defaults to unset.`

This also needs to describe how to encode IPv6 numbers, since : is used in them too.
An example would be great.